frontend/pulse3d/src/pages/uploads.js

```diff
@@ -640,6 +650,20 @@ export default function Uploads() {
               }
               onSelectedRowsChange={({ selectedRows, selectedCount }) => {
                 let arr = [];
+                if (selectedRows.length > 0) {
+                  selectedRows.forEach((row) => {
+                    row.jobs.forEach((job) => {
+                      if (!job.checked) {
+                        for (let i = 0; i < jobs.length; i++) {
+                          if (jobs[i].jobId === job.jobId) {
+                            jobs[i].checked = true;
+                            setCheckedJobs([...checkedJobs, jobs[i].jobId]);
+                          }
+                        }
+                      }
+                    });
+                  });
+                }
```

Comment:
Say a user selects an upload to check all the corresponding jobs, does this include a way to uncheck the upload row if a user unchecks one of the selected jobs? and then a way to uncheck all jobs if a user unchecks an upload row?

Reply:
No not yet. I will add this